packages/e/eopkg/package.yml

```diff
@@ -0,0 +1,132 @@
+name       : eopkg
+version    : 4.0.0
+release    : 6
```

Comment:
Initial release can be 4 to safely update python-eopkg

Reply:
Ack.